.github/workflows/testBuild.yml

```diff
@@ -193,7 +216,7 @@ jobs:
           ":test_tube::test_tube: Use the links below to test this build in android and iOS. Happy testing! :test_tube::test_tube:
           | android :robot:  | iOS :apple: | desktop :computer: |
           | ------------- | ------------- | ------------- | 
-          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  | https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/${{github.event.number}}/NewExpensify.dmg |
-          | ![Android](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=${{fromJson(steps.set_var.outputs.android_paths).html_path}}) | ![iOS](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=${{fromJson(steps.set_var.outputs.ios_paths).html_path}}) | ![desktop](https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/${{github.event.number}}/NewExpensify.dmg) |"
+          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  | https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/$BRANCH_HASH/NewExpensify.dmg |
```

Comment:
Can we just use the PR number here instead? I think it would be nice to have some ID that we can relate back to the PR instead of a random-ish hash.

Reply:
Sure, no problem! I've added a small check to make sure that the PR number that is in the input matches the chosen branch. :)